static/js/vue-cdr-access/src/mixins/displayUtils.js

```diff
@@ -0,0 +1,21 @@
+export default {
+    methods: {
+        recordUrl(id) {
+            return `/record/${id}`;
+        },
+
+        recordType(type) {
+            if (type === 'Collection') {
+                return 'fa-archive';
+            } else if (type === 'Folder') {
+                return 'fa-folder';
+            } else {
+                return 'fa-file';
+            }
+        },
+
+        hasThumbnail(thumb) {
```

Comment:
this method is basically a "is empty" check. I'd suggest either renaming it, or passing in a record instead and having it pull out the `thumbnail_url` property